packages/autocomplete/index.ts

```diff
@@ -1,8 +1,11 @@
 import { App } from 'vue'
+import { SFCWithInstall } from '../utils/types'
 import Autocomplete from './src/index.vue'
 
 Autocomplete.install = (app: App): void => {
   app.component(Autocomplete.name, Autocomplete)
 }
 
-export default Autocomplete
+const _Autocomplete: SFCWithInstall<typeof Autocomplete> = Autocomplete
+
+export default _Autocomplete
```

Comment:
Can we do something like this? Rather than reassigning it, which is totally meaningless statement.
![image](https://user-images.githubusercontent.com/15975785/102773884-ea63eb80-43c4-11eb-835a-6f05a0ede22e.png)

Reply:
Using `cast` is better than the way you did IMHO.

Reply:
Using `cast` seems to generate the d.ts file incorrectly. It's missing `import Comp from 'path'` and the inclusion of all the types.

![image](https://user-images.githubusercontent.com/25665423/102841412-e0330300-443f-11eb-8f9b-a09ad28fce1b.png)

Reply:
I don't think it's a good idea to reassign values either, but I haven't found any other way.
And i don't really know how tsc generates d.ts file.

Reply:
Seems reasonable to me.